src/components/Collections/Table/Table.vue

```diff
@@ -206,11 +202,12 @@ export default {
       // classes for search bar input box
       searchInputBoxClass:
         "w-full text-gray-700 leading-tight p-2 pl-4 focus:outline-none",
+      // sort order for the "number of viewers" column. 1 - ascending, -1 - descending
+      numberOfViewersSortOrder: 1,
```

Comment:
seems too too long to me (generally, Of, For like words are not present in variable names)

```suggestion
      numViewersSortOrder: 1,
```